src/main/java/training/Shapes

```diff
@@ -0,0 +1,70 @@
+package training;
```

Comment:
Yes this looking good now you are able to use both polymorphism and inheritance.